linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/routers.js

```diff
@@ -36,12 +36,31 @@ var Routers = (function() {
   var clientRE = /^rt\/(.+)\/dst\/id\/(.*)\/requests$/,
       serverRE = /^rt\/(.+)\/srv\/([^/]+)\/(\d+)\/requests$/;
 
+  var mkColor = function() {
+    var colors = [
+      [152,224,197],
+      [74,216,172],
+      [64,118,196],
+      [83,176,196],
+      [166,220,0],
+      [0,0,43]
+    ];
```

Comment:
I think I'm just missing this in the diff, but it seems like wherever you're using a color value you're always calling `.join(",")` -- why not just making this an array of strings to start with?